blocks/featured-resort-cards/featured-resort-cards.js

```diff
@@ -0,0 +1,65 @@
+import {
+  div, domEl, p, a, img,
+} from '../../scripts/dom-helpers.js';
+
+export default async function decorate(block) {
+  const dom = domEl('ul');
+  dom.className = 'resorts-grid-section';
+
+  const mapPinPath = '../../icons/mapPin.svg';
+  const chevronRight = '../../icons/chevronRight.svg';
+
+  [...block.children].forEach((row, index) => {
+    // skip the first element, which is used to inform the authors the column structure
+    if (index === 0) {
+      return;
+    }
+
+    const resortPicture = row.children[0].querySelector('picture');
+    const resortTitle = row.children[1];
+    const resortDescription = row.children[2];
+    const viewMapLink = row.children[3].querySelector('p').querySelector('a').href;
+    const viewMapTitle = row.children[3].querySelector('p').querySelector('a').title;
+    const viewAccommodationsLink = row.children[4].querySelector('p').querySelector('a').href;
+    const viewAccommodationsTitle = row.children[4].querySelector('p').querySelector('a').title;
+
+    const il = domEl(
+      'li',
+      div(
+        { class: 'main-resort-card' },
+        a(
+          { href: '#' },
```

Comment:
This link probably needs to go somewhere, add as an author-able item? Rows are getting kinda full 🤔